index.bs

```diff
@@ -633,6 +637,21 @@ The information about position and orientation of the intersection point should
 
 Issue: Decide if we need to specify other axes of the coordinate system defined by hit test result's native origin to maintain compatibility between different implementations & differrent AR frameworks.
 
+Privacy & Security Considerations {#privacy-security}
+=================================
+
+<section class="non-normative">
+
+The hit test API can be used to map the user environment with more or less
+precision by sending hit test rays in multitude directions. Hit test has to be
```

Comment:
s/"multitude"/"multiple", or s/"multitude"/"multitude of" ?

Reply:
Done.